backend/src/core/middlewares/job.middleware.ts

```diff
@@ -98,8 +116,28 @@ const retryCampaign = async (
   }
 }
 
+const cancelScheduledCampaign = async (
+  req: Request,
+  res: Response,
+  next: NextFunction
+): Promise<Response | void> => {
+  try {
+    const { campaignId } = req.params
+    const logMeta = { campaignId, action: 'cancelScheduledCampaign' }
+    logger.info({
+      message: 'Cancel Scheduled Campaign...',
```

Comment:
nit, but would suggest removing the ellipsis to keep the style consistent

Reply:
removing which ellipsis?

Reply:
oh the `...` haha just a nit